Siddhi build fails on Windows OS

**Description:**
Siddhi build fails on Windows OS throwing checkstyle errors. However building in Ubuntu in a virtual machine on the same PC doesn't throw such errors.

[INFO] ------------------------------------------------------------------------
[INFO] Building Siddhi Query API 4.0.0-M117-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- maven-checkstyle-plugin:2.17:check (validate) @ siddhi-query-api ---
[INFO] Starting audit...
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\aggregation\TimePeriod.java:108: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\annotation\Annotation.java:25: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\annotation\Element.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\definition\AbstractDefinition.java:28: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\definition\AggregationDefinition.java:30: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\definition\Attribute.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\definition\Attribute.java:102: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\definition\FunctionDefinition.java:23: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\definition\StreamDefinition.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\definition\TriggerDefinition.java:24: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\definition\WindowDefinition.java:31: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\exception\AttributeNotExistException.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\exception\DuplicateAnnotationException.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\exception\DuplicateAttributeException.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\exception\DuplicateDefinitionException.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\exception\SiddhiAppContextException.java:21: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\exception\SiddhiAppValidationException.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\exception\UnsupportedAttributeTypeException.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\ExecutionElement.java:24: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\partition\PartitionType.java:23: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\partition\RangePartitionType.java:25: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\partition\RangePartitionType.java:107: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\partition\ValuePartitionType.java:23: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\handler\Filter.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\handler\StreamFunction.java:26: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\handler\StreamHandler.java:23: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\handler\Window.java:26: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\state\CountStateElement.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\state\EveryStateElement.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\state\LogicalStateElement.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\state\LogicalStateElement.java:120: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\state\NextStateElement.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\state\State.java:24: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\state\StateElement.java:23: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\state\StreamStateElement.java:23: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\store\AggregationInputStore.java:24: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\store\ConditionInputStore.java:23: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\store\InputStore.java:21: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\store\Store.java:25: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\stream\AnonymousInputStream.java:27: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\stream\BasicSingleInputStream.java:28: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\stream\InputStream.java:28: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\stream\JoinInputStream.java:26: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\stream\JoinInputStream.java:170: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\stream\JoinInputStream.java:181: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\stream\SingleInputStream.java:30: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\stream\StateInputStream.java:30: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\input\stream\StateInputStream.java:140: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\ratelimit\EventOutputRate.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\ratelimit\OutputRate.java:27: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\ratelimit\OutputRate.java:81: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\ratelimit\SnapshotOutputRate.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\ratelimit\TimeOutputRate.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\stream\DeleteStream.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\stream\InsertIntoStream.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\stream\OutputStream.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\stream\OutputStream.java:105: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\stream\ReturnStream.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\stream\UpdateOrInsertStream.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\stream\UpdateSet.java:28: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\stream\UpdateSet.java:94: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\output\stream\UpdateStream.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\Query.java:38: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\selection\BasicSelector.java:25: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\selection\OutputAttribute.java:24: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\selection\Selector.java:28: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\execution\query\StoreQuery.java:24: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\AttributeFunction.java:24: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\condition\And.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\condition\Compare.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\condition\Compare.java:94: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\condition\In.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\condition\IsNull.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\condition\Not.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\condition\Or.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\constant\BoolConstant.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\constant\Constant.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\constant\DoubleConstant.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\constant\FloatConstant.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\constant\IntConstant.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\constant\LongConstant.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\constant\StringConstant.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\constant\TimeConstant.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\Expression.java:41: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\Expression.java:168: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\math\Add.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\math\Divide.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\math\Mod.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\math\Multiply.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\math\Subtract.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\expression\Variable.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\extension\Extension.java:22: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\SiddhiApp.java:43: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\SiddhiElement.java:24: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\util\ExceptionUtil.java:23: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\main\java\org\wso2\siddhi\query\api\util\SiddhiConstants.java:20: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\DefineAggregationTestCase.java:33: error: Missing a Javadoc comment.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\DefineStreamTestCase.java:27: error: Missing a Javadoc comment.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\DefineTableTestCase.java:26: error: Missing a Javadoc comment.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\DefineTriggerTestCase.java:29: error: Missing a Javadoc comment.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\JoinQueryTestCase.java:31: error: Missing a Javadoc comment.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\OutputRateTestCase.java:28: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\PartitionQueryTestCase.java:29: error: Missing a Javadoc comment.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\PatternQueryTestCase.java:31: error: Missing a Javadoc comment.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\SequenceQueryTestCase.java:29: error: First sentence should end with a period.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\SimpleQueryTestCase.java:29: error: Missing a Javadoc comment.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\StoreQueryTestCase.java:29: error: Missing a Javadoc comment.
E:\WSO2\siddhi\modules\siddhi-query-api\src\test\java\org\wso2\siddhi\query\api\TableQueryTestCase.java:30: error: Missing a Javadoc comment.
Audit done.
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Siddhi ............................................. SUCCESS [  7.206 s]
[INFO] Siddhi Query API ................................... FAILURE [  1.511 s]
[INFO] Siddhi Query Compiler .............................. SKIPPED
[INFO] Siddhi Annotations ................................. SKIPPED
[INFO] Siddhi Core ........................................ SKIPPED
[INFO] Siddhi As a Service ................................ SKIPPED
[INFO] Siddhi Samples ..................................... SKIPPED
[INFO] Siddhi Performance Samples ......................... SKIPPED
[INFO] Siddhi Quick Start Samples ......................... SKIPPED
[INFO] Siddhi Documentation Generator ..................... SKIPPED
[INFO] Siddhi Core Documentation Generation ............... SKIPPED
[INFO] Siddhi Tests Coverage Report Aggregation ........... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 10.134 s
[INFO] Finished at: 2017-10-27T12:52:42+05:30
[INFO] Final Memory: 23M/56M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (validate) on project siddhi-query-api: Failed during checkstyle execution: There are 108 errors reported by
Checkstyle 6.11.2 with https://raw.githubusercontent.com/wso2/code-quality-tools/master/checkstyle/checkstyle.xml ruleset. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :siddhi-query-api

**Affected Product Version:**
Current repo. However I encountered this with previous versions also

**OS, DB, other environment details and versions:**    
OS: Windows 10 Education version: 1703
Hardware: MSI GP62 7REX

**Steps to reproduce:**
Try building Siddhi in Windows 10 OS

--- a/checkstyle-suppressions.xml
+++ b/checkstyle-suppressions.xml
@@ -23,25 +23,25 @@
 
 <suppressions>
 
-    <suppress checks="Javadoc.*" files=".*/src/test/java/.*"/>
-    <suppress checks=".*" files=".*/antlr4/org/wso2/siddhi/query/compiler/*" />
-    <suppress checks=".*" files=".*/src/gen/java/org/wso2/siddhi/service/*" />
+    <suppress checks="Javadoc.*" files=".*[/\\]src[/\\]test[/\\]java[/\\].*"/>
+    <suppress checks=".*" files=".*[/\\]antlr4[/\\]org[/\\]wso2[/\\]siddhi[/\\]query[/\\]compiler[/\\]*" />
+    <suppress checks=".*" files=".*[/\\]src[/\\]gen[/\\]java[/\\]org[/\\]wso2[/\\]siddhi[/\\]service[/\\]*" />
 
-    <suppress checks="JavadocPackage" files=".*/src/(main|integration)/java/.*"/>
-    <suppress checks="JavadocPackage" files=".*/src/.*/internal/.*"/>
+    <suppress checks="JavadocPackage" files=".*[/\\]src[/\\](main|integration)[/\\]java[/\\].*"/>
+    <suppress checks="JavadocPackage" files=".*[/\\]src[/\\].*[/\\]internal[/\\].*"/>
 
-    <suppress checks="JavadocStyle" files=".*/src/(main|integration)/java/.*"/>
-    <suppress checks="JavadocStyle" files=".*/src/.*/internal/.*"/>
+    <suppress checks="JavadocStyle" files=".*[/\\]src[/\\](main|integration)[/\\]java[/\\].*"/>
+    <suppress checks="JavadocStyle" files=".*[/\\]src[/\\].*[/\\]internal[/\\].*"/>
 
     <!--do not check antler files -->
-    <suppress checks="MemberNameCheck" files=".*/internal/SiddhiQLBaseVisitorImpl.java"/>
-    <suppress checks="ParameterName" files=".*/internal/SiddhiQLBaseVisitorImpl.java"/>
-    <suppress checks="LocalVariableName" files=".*/internal/SiddhiQLBaseVisitorImpl.java"/>
-    <suppress checks="LocalFinalVariableName" files=".*/internal/SiddhiQLBaseVisitorImpl.java"/>
-    <suppress checks="Regexp" files=".*/debugger/SiddhiDebuggerClient.java"/>
-    <suppress checks="Regexp" files=".*/siddhi/performance/.*"/>
-    <suppress checks="Javadoc.*" files=".*/siddhi/performance/.*"/>
-    <suppress checks="Regexp" files=".*/siddhi/sample/.*"/>
-    <suppress checks="Javadoc.*" files=".*/siddhi/sample/.*"/>
+    <suppress checks="MemberNameCheck" files=".*[/\\]internal[/\\]SiddhiQLBaseVisitorImpl.java"/>
+    <suppress checks="ParameterName" files=".*[/\\]internal[/\\]SiddhiQLBaseVisitorImpl.java"/>
+    <suppress checks="LocalVariableName" files=".*[/\\]internal[/\\]SiddhiQLBaseVisitorImpl.java"/>
+    <suppress checks="LocalFinalVariableName" files=".*[/\\]internal[/\\]SiddhiQLBaseVisitorImpl.java"/>
+    <suppress checks="Regexp" files=".*[/\\]debugger[/\\]SiddhiDebuggerClient.java"/>
+    <suppress checks="Regexp" files=".*[/\\]siddhi[/\\]performance[/\\].*"/>
+    <suppress checks="Javadoc.*" files=".*[/\\]siddhi[/\\]performance[/\\].*"/>
+    <suppress checks="Regexp" files=".*[/\\]siddhi[/\\]sample[/\\].*"/>
+    <suppress checks="Javadoc.*" files=".*[/\\]siddhi[/\\]sample[/\\].*"/>
 
 </suppressions>

